akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/directives/RespondWithDirectivesExamplesSpec.scala

```diff
@@ -80,68 +80,4 @@ class RespondWithDirectivesExamplesSpec extends RoutingSpec {
     }
   }
 
-  //  FIXME awaiting resolution of https://github.com/akka/akka/issues/18625
-  //  "respondWithMediaType-examples" in {
-  //    import MediaTypes._
-  //
-  //    val route =
-  //      path("foo") {
-  //        respondWithMediaType(`application/json`) {
-  //          complete("[]") // marshalled to `text/plain` here
-  //        }
-  //      }
-  //
-  //    Get("/foo") ~> route ~> check {
-  //      mediaType shouldEqual `application/json`
-  //      responseAs[String] shouldEqual "[]"
-  //    }
-  //
-  //    Get("/foo") ~> Accept(MediaRanges.`text/*`) ~> route ~> check {
-  //      rejection shouldEqual UnacceptedResponseContentTypeRejection(ContentType(`application/json`) :: Nil)
-  //    }
-  //  }
-
-  //  "respondWithSingletonHeader-examples" in {
-  //    val respondWithMuppetHeader =
-  //      respondWithSingletonHeader(RawHeader("Funky-Muppet", "gonzo"))
-  //
-  //    val route =
-  //      path("foo") {
-  //        respondWithMuppetHeader {
-  //          complete("beep")
-  //        }
-  //      } ~
-  //        path("bar") {
-  //          respondWithMuppetHeader {
-  //            respondWithHeader(RawHeader("Funky-Muppet", "kermit")) {
-  //              complete("beep")
-  //            }
-  //          }
-  //        }
-  //
-  //    Get("/foo") ~> route ~> check {
-  //      headers.filter(_.is("funky-muppet")) shouldEqual List(RawHeader("Funky-Muppet", "gonzo"))
-  //      responseAs[String] shouldEqual "beep"
-  //    }
-  //
-  //    Get("/bar") ~> route ~> check {
-  //      headers.filter(_.is("funky-muppet")) shouldEqual List(RawHeader("Funky-Muppet", "kermit"))
-  //      responseAs[String] shouldEqual "beep"
-  //    }
-  //  }
-
-  // FIXME https://github.com/akka/akka/issues/18626
-  //  "respondWithStatus-examples" in {
```

Comment:
these directives are not ported as deemed as anti-patterns, documented below and will be in  the full blown migration guide